### PR TITLE
parameter_validation_generator: Add support for arraylength implicit VUs

### PR DESCRIPTION
**Pull request by JasperNV**

### CODE REVIEW

File: layers/stateless_validation.h [hunk 1]
```diff
@@ -303,12 +303,12 @@ class StatelessValidation : public ValidationObject {
     bool validate_struct_type_array(const char *apiName, const ParameterName &countName, const ParameterName &arrayName,
                                     const char *sTypeName, uint32_t count, const T *array, VkStructureType sType,
                                     bool countRequired, bool arrayRequired, const std::string &stype_vuid,
-                                    const std::string &param_vuid) {
+                                    const std::string &param_vuid, const std::string& count_required_vuid) {
```
Comment (jeffbolznv) [inline layers/stateless_validation.h]: Please change to "const char *" (see https://github.com/KhronosGroup/Vulkan-ValidationLayers/pull/667).

Comment (JasperNV) [reply layers/stateless_validation.h]: That's going to cause a merge conflict, it looks like. Let's park this until that PR goes through.

**Comment by JasperNV** [comment]
Rebased on top of Jeff's string changes.

**Comment by mark-lunarg** [comment]
@jzulauf-lunarg, are you running this through CI?  If not lemme know and I'll fire it off...

**Comment by mark-lunarg** [comment]
Running through the farm now.

**Comment by mark-lunarg** [comment]
Passed on the farm, and we picked up 14 VUs!  Pushed to Master.  Thanks @JasperNV...

**Pull request closed without merging**